cmds/copy.c

```diff
@@ -127,15 +132,15 @@ static int cmd_copy(char *s)
 		return -EINVAL;
 	}
 
-	if (cmd_parseDev(&h[0], &offs[0], &sz[0], args, &argsID, argsc) < 0)
+	if (cmd_parseDev(&h[0], &offs[0], &sz[0], args, &argsID, argsc, &file[0]) < 0)
 		return -EINVAL;
 
-	if (cmd_parseDev(&h[1], &offs[1], &sz[1], args, &argsID, argsc) < 0)
+	if (cmd_parseDev(&h[1], &offs[1], &sz[1], args, &argsID, argsc, &file[1]) < 0)
 		return -EINVAL;
 
 	/* Copy data between devices */
 	log_info("\nCopying data, please wait...");
-	if (cmd_cpphfs2phfs(h[0], offs[0], sz[0], h[1], offs[1], sz[1]) < 0) {
+	if ((res = cmd_cpphfs2phfs(h[0], offs[0], sz[0], h[1], offs[1], sz[1])) < 0) {
```

Comment:
If you are using `res` variable here then you can simplify the code to:
```
	res = cmd_cpphfs2phfs(h[0], offs[0], sz[0], h[1], offs[1], sz[1]);

	phfs_close(h[0]);
	phfs_close(h[1]);

	if (res < 0) {
		log_error("\nCopying failed");
		return -EINVAL;
	}

	log_info("\nFinished copying");
```

Reply:
Done